### PR TITLE
add suggestedValue

### PR DESCRIPTION
**Pull request by gary-codaio**
Rename defaultValue to suggestedValue. Together with https://github.com/coda/coda/pull/70522

### CODE REVIEW

File: CHANGELOG.md
Comment (jonathan-codaio) [inline CHANGELOG.md:14]: Nit, can hold off on the May 1 language for now, since we'll bundle all these changes together we can put a date on all of them at once.

File: helpers/external-api/v1.ts
Comment (jonathan-codaio) [inline helpers/external-api/v1.ts:756]: This is a generated code file copied over from `coda`, and confusingly this field actually refers something different. So can just revert this file.

File: helpers/migration.ts
Comment (jonathan-codaio) [inline helpers/migration.ts:65]: Nit, the approach I was taking with the other helpers was just to have one way of referring to the value, i.e. eliminate this redundant getter. Then the changes in code that references these fields is simply wrapping the parameter access with the helper, leaving the field name unchanged, hopefully making it easy to review the changes for correctness and safety.

File: testing/upload_validation.ts
Comment (jonathan-codaio) [inline testing/upload_validation.ts:444]: Hrm weird, I guess this should have been optional the whole time. I guess `unknown()` was already permitting optional values implicitly?

**Review by jonathan-codaio — COMMENTED** [review]
Nice

File: testing/upload_validation.ts
Comment (gary-codaio) [reply testing/upload_validation.ts:444]: changed it back to `unknown()`

Comment (jonathan-codaio) [reply testing/upload_validation.ts:444]: Well, was thinking they should all probably have `.optional()` on there for clarity, even if redundant.

Comment (gary-codaio) [reply testing/upload_validation.ts:444]: Oh gotcha, then in that case, will add it back along with the other unknowns here

**Review by jonathan-codaio — APPROVED** [review]
After you merge this, you'll want to take the post-merge commit hash, and update your `coda` PR with that hash (since the commit hash from this branch will disappear once the branch is deleted). You'll want to have the `coda` side change queued up to merge reasonably soon after merging this, so other's with SDK changes after yours don't get stuck.

**Pull request merged**
